src/RapidezStatamicServiceProvider.php

```diff
@@ -31,26 +31,27 @@ public function boot()
             ->bootConfig()
             ->bootRepositories()
             ->bootPublishables()
-            ->bootFilters()
             ->bootComposers()
-            ->bootListeners();
+            ->bootListeners()
+            ->bootRoutes();
     }
 
-    public function bootConfig() : self
+    public function bootRoutes() : self
     {
-        $this->mergeConfigFrom(__DIR__.'/../config/statamic.php', 'statamic');
+        if (!$this->app->routesAreCached()) {
```

Comment:
Maybe this is something for every rapidez package. Because it is unnecessary to load routes when they're cached.

Reply:
Indeed, PR bomb incoming... 😋